File: app/src/main/java/com/shahu/weathery/model/Wind.java
package com.shahu.weathery.model;

import javax.annotation.Generated;
import com.google.gson.annotations.SerializedName;

@Generated("com.robohorse.robopojogenerator")
public class Wind{

	@SerializedName("deg")
	private int deg;

	@SerializedName("speed")
	private double speed;

	public void setDeg(int deg){
		this.deg = deg;
	}

	public int getDeg(){
		return deg;
	}

	public void setSpeed(double speed){
		this.speed = speed;
	}

	public double getSpeed(){
		return speed;
	}

	@Override
 	public String toString(){
		return 
			"Wind{" + 
			"deg = '" + deg + '\'' + 
			",speed = '" + speed + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/shahu/weathery/model/main/Wind.kt
package com.shahu.weathery.model.main

import com.google.gson.annotations.SerializedName
import javax.annotation.Generated

@Generated("com.robohorse.robopojogenerator")
class Wind {
    @SerializedName("deg")
    var deg = 0

    @SerializedName("speed")
    var speed = 0.0

    override fun toString(): String {
        return "Wind{" +
                "deg = '" + deg + '\'' +
                ",speed = '" + speed + '\'' +
                "}"
    }
}